packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/code/Code.java

```diff
@@ -54,7 +48,8 @@ public class Code {
 	public enum ProgrammingLanguage {
 		PYTHON("python"),
 		R("r"),
-		Julia("julia");
+		Julia("julia"),
+		ZIP("zip");
```

Comment:
I know zip is not a language, but i couldn't find a better way to represent zips for now...Im sure this will be fixed when we add the file tree